models/web/schema.yml

```diff
@@ -0,0 +1,379 @@
+      - name: br_name
+        description: '{{ doc("col_br_name") }}'
+      - name: dvce_type
+        description: '{{ doc("col_dvce_type") }}'
+      - name: os_name
+        description: '{{ doc("col_os_name") }}'
+      - name: os_timezone
+        description: '{{ doc("col_os_timezone") }}'
+      - name: start_tstamp
+        description: '{{ doc("col_start_tstamp") }}'
+      - name: end_tstamp
+        description: '{{ doc("col_end_tstamp") }}'
+      - name: play_time_sec
+        description: '{{ doc("col_play_time_sec") }}'
+      - name: play_time_sec_muted
+        description: '{{ doc("col_play_time_sec_muted") }}'
+      - name: is_played
+        description: '{{ doc("col_is_played") }}'
+      - name: is_valid_play
+        description: '{{ doc("col_is_valid_play") }}'
+      - name: is_complete_play
+        description: '{{ doc("col_is_complete_play") }}'
+      - name: avg_playback_rate
+        description: '{{ doc("col_avg_playback_rate") }}'
+      - name: retention_rate
+        description: '{{ doc("col_retention_rate") }}'
+      - name: seeks
+        description: '{{ doc("col_seeks") }}'
+      - name: percent_progress_reached
+        description: '{{ doc("col_percent_progress_reached") }}'
+  - name: snowplow_media_player_plays_by_session
+    description: '{{ doc("table_plays_by_session") }}'
+    columns:
+      - name: domain_sessionid
+        description: '{{ doc("col_domain_sessionid") }}'
+        tags:
+          - primary-key
+        tests:
+          - unique
+          - not_null
+      - name: domain_userid
+        description: '{{ doc("col_domain_userid") }}'
+      - name: impressions
+        description: '{{ doc("col_impressions") }}'
+      - name: videos_played
+        description: '{{ doc("col_videos_played") }}'
+      - name: audio_played
+        description: '{{ doc("col_audio_played") }}'
+      - name: video_plays
+        description: '{{ doc("col_video_plays") }}'
+      - name: audio_plays
+        description: '{{ doc("col_audio_plays") }}'
+      - name: valid_video_plays
+        description: '{{ doc("col_valid_video_plays") }}'
+      - name: valid_audio_plays
+        description: '{{ doc("col_valid_audio_plays") }}'
+      - name: start_tstamp
+        description: '{{ doc("col_start_tstamp") }}'
+      - name: end_tstamp
+        description: '{{ doc("col_end_tstamp") }}'
+      - name: seeks
+        description: '{{ doc("col_seeks") }}'
+      - name: play_time_sec
+        description: '{{ doc("col_play_time_sec") }}'
+      - name: play_time_sec_muted
+        description: '{{ doc("col_play_time_sec_muted") }}'
+      - name: avg_play_time_sec
+        description: '{{ doc("col_avg_play_time_sec") }}'
+      - name: avg_percent_played
+        description: '{{ doc("col_avg_percent_played") }}'
+      - name: complete_plays
+        description: '{{ doc("col_complete_plays") }}'
+  - name: snowplow_media_player_user_stats
```

Comment:
You should remove this table definition and tests from here and add it to the `custom` folder